src/FactoryGenerator.php

```diff
@@ -68,7 +68,11 @@ public function generate($model)
 
     private function factoryPath($model)
     {
-        return database_path('factories/' . class_basename($model) . 'Factory.php');
+        $subDirectory = Str::of($model)
+            ->replaceFirst('App\\Models\\', '')
```

Comment:
I am going to merge this. However, it's still not a very good check. Consider what would happen with `Custom\\Namespace\\App\\Models\\Foo`...